src/index.ts

```diff
@@ -34,73 +38,30 @@ export async function execute(filenames: string[], options: Partial<ExecutionOpt
     const executionOptions = Object.assign(defaultExecutionOptions, options);
 
     let manifest = await PP.getManifest(executionOptions.root);
+    let componentDefinitions = await getComponentDefinitionMap(executionOptions.root);
+
+    componentDefinitions.forEach((component: ComponentDefinition) => {
+        if (component.scripts.length < 1) return;
+        component.scripts = component.scripts.map((script: ComponentScript) => {
+            script.uri = path.join(
+                options.root ? options.root : __dirname,
```

Comment:
I think I opted to use `__dirname` over `process.cwd()` since you could run this from a different directory. It seems like that would be wrong choice in that situation?